addons/sale/security/sale_security.xml

```diff
@@ -137,6 +137,34 @@
         <field name="groups" eval="[(4, ref('sales_team.group_sale_salesman'))]"/>
     </record>
 
+    <record id="account_move_personal_rule" model="ir.rule">
+        <field name="name">Personal Invoice</field>
+        <field ref="model_account_move" name="model_id"/>
+        <field name="domain_force">[('type','in',('out_invoice', 'out_refund', 'in_invoice', 'in_refund')), '|', ('invoice_user_id', '=', user.id), ('invoice_user_id', '=', user.id)]</field>
+        <field name="groups" eval="[(4, ref('sales_team.group_sale_salesman'))]"/>
+    </record>
+
+    <record id="account_move_see_all" model="ir.rule">
+        <field name="name">All Invoices</field>
+        <field ref="model_account_move" name="model_id"/>
+        <field name="domain_force">[(1,'=',1)]</field>
```

Comment:
shouldn't there be a restriction on `type`?

Reply:
Indeed